api/v1beta1/hyperconverged_types.go

```diff
@@ -205,8 +205,6 @@ type HyperConvergedSpec struct {
 	VMStateStorageClass *string `json:"vmStateStorageClass,omitempty"`
 
 	// VirtualMachineOptions holds the cluster level information regarding the virtual machine.
-	// +kubebuilder:default={"disableFreePageReporting": true}
-	// +default={"disableFreePageReporting": true}
```

Comment:
I'm not so sure that this will be enough for who initially deployed with the previous release and now upgrades.
What are we supposed to do in that case?

Reply:
@tiraboschi Thank you for the review! I would expect that during operator rollout, any change in defaults will take effect, unless the field was modified by the user. Does HCO have such mechanism?